interop-tests/src/test/java/org/apache/pekko/grpc/interop/PekkoGrpcJavaClientTester.java

```diff
@@ -34,49 +35,72 @@
 import io.grpc.testing.integration.UnimplementedServiceClient;
 import io.grpc.testing.integration2.ClientTester;
 import io.grpc.testing.integration2.Settings;
-import org.junit.Assert;
-import scala.concurrent.ExecutionContext;
 
 import java.io.InputStream;
+
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
 import java.util.concurrent.CompletionStage;
 import java.util.concurrent.TimeUnit;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+
+import scala.concurrent.ExecutionContext;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.fail;
 
 /**
  * ClientTester implementation that uses the generated pekko-grpc Java client to exercise a server under test.
- *
+ * <p>
```

Comment:
minor: is this needed?

Reply:
thanks.
 Since pekko grpc did not use Javafmt before, I used idea formatting, which usually recommends adding <p> on empty lines
 
I have reformatted by javafmt and converted it, but it has not been pushed yet,  I am testing the PR.